README.rst

```diff
@@ -22,6 +22,108 @@ For more options, see::
 
     $ make help
 
+Configure using thinkhazard_processing.yaml
+===========================================
+
+Keys in configuration file:
+
+sqlalchemy.url
+--------------
+
+Database connection parameters, example:
+
+.. code:: yaml
+
+    sqlalchemy.url: postgresql://www-data:www-data@localhost:5432/thinkhazard_processing
+
+data_path
+---------
+
+Path to main data folder, example:
+
+.. code:: yaml
+
+    data_path: /var/sig
+
+For production, we recommend a dedicated disk partition.
+
+hazard_types
+------------
+
+Harvesting and processing configuration for each hazard type.
+One entry for each hazard type mnemonic, with subkeys:
+
+- ``hazard_type``: Corresponding hazard_type value in geonode.
+
+- ``preprocessed``: Mandatory processing method. Possible values are:
```

Comment:
I don't see the use of ```preprocessed```.
If the admin provides a ```values``` block, then we should consider the hazardtype as preprocessed.
If not, there should be ```return_periods``` and ```thresholds```.